src/controllers/loader.py

```diff
@@ -0,0 +1,63 @@
+import os
+import dotenv
+import yaml
+from pydantic import Field, validator
+from pydantic_settings import BaseSettings
+from typing import Dict, Tuple, List
+
+from src.models.loader import LoaderConfig, FileLoader
+from src.models.env_config import get_config
+from src.constants import LOADER_CONFIG_FILE
+
+
+class LoaderManager:
```

Comment:
We can just implement File loader config but introducing an abstract config to manage the loader.yaml is not a bad idea.